Resources/Private/Language/locallang.xlf

```diff
@@ -3,8 +3,11 @@
     <file source-language="en" datatype="plaintext" original="EXT:jobapplications/Resources/Private/Language/locallang" date="2019-10-21T16:03:17Z" product-name="jobapplications">
         <header/>
         <body>
+            <trans-unit id="tx_jobapplications_domain_model_posting.date_of_application" resname="tx_jobapplications_domain_model_posting.date_of_application">
+                <source>Date of Application</source>
+            </trans-unit>
             <trans-unit id="date_format" resname="date_format">
-                <source>m-d-Y</source>
+                <source>d.m.Y</source>
```

Comment:
@itx-patrick-heinz Diese Änderung bitte als Übersetzung in de.locallang.xlf packen